CONTRIBUTORS

```diff
@@ -0,0 +1,36 @@
+Bleach was originally written and maintained by Erik Rose and various
+contributors within and without the Mozilla Corporation and Foundation.
+
+It is currently maintained by XYZ.
```

Comment:
```suggestion
It is currently maintained by Hood Chatham.
```
Or I can insert this myself later. Thanks!

Reply:
Done!